store/policy.go

```diff
@@ -164,6 +164,24 @@ func (s *Store) GetBackupPlanPolicyByEnvID(ctx context.Context, environmentID in
 	return api.UnmarshalBackupPlanPolicy(policy.Payload)
 }
 
+// SomeDatabaseEnablesBackupPolicyInInstance returns whether at least one of the databases in the given instance enables backup policy.
+func (s *Store) SomeDatabaseEnablesBackupPolicyInInstance(ctx context.Context, instanceID int) (bool, error) {
+	var id int
+	if err := s.db.db.QueryRowContext(ctx, `
+		SELECT db.id
+		FROM db
```

Comment:
As a second thought, can we just return the list of MYSQL instances rather than doing branching in both code and query?